packages/hd-transport-web-device/src/electron-ble-transport.ts

```
@@ -0,0 +1,365 @@
+import transport, { COMMON_HEADER_SIZE, LogBlockCommand } from '@onekeyfe/hd-transport';
+import {
+  ERRORS,
+  HardwareErrorCode,
+  Deferred,
+  createDeferred,
+  isHeaderChunk,
+} from '@onekeyfe/hd-shared';
+import ByteBuffer from 'bytebuffer';
+import type EventEmitter from 'events';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)